components/BreadCrumbs.vue

```diff
@@ -20,9 +28,15 @@ export default {
       if (crumbLocations[i - 1]) {
         nextName = ( `${ crumbLocations[i - 1].name }-${ piece }`);
       }
+      const nextParams = this.paramsFor(nextName, params);
+
+      if (nextName === 'c-cluster-resource' && params.resource === EXTERNAL.PROJECT) {
+        nextParams.resource = NAMESPACE;
+      }
```

Comment:
I don't get what this is accomplishing?  Your screenshots show editing a project but the breadcrumb saying namespace; and the type missing entirely in `Edit <type>: <id>` header.

Reply:
I don't think it's necessary. I suspect it was exploratory code while modifying this so that it shows namespace when in a project detail page since we don't have a project list page.

The type missing is completely unrelated to this. That was caused by masthead looking up the schema for project in the wrong module. I also made that change.

Reply:
Actually it wasn't exploratory. It was to change the link to link to namespace instead of project. The other change in here just changes what's displayed not what's linked.